### PR TITLE
[prometheus-statsd-exporter] add first release

### PR DESCRIPTION
**Pull request by scDisorder**
Signed-off-by: disorder <disordervlz@gmail.com>

#### What this PR does / why we need it:
- I needed one, so I made it. Just though to share it will be nice.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by scDisorder** [comment]
hmm, probably the issue that helm actually does not runs without mappings specified...
wait a minute

**Comment by torstenwalter** [comment]
Can you add yourself to CODEOWNERS file?

File: charts/prometheus-statsd-exporter/README.md
Comment (torstenwalter) [inline charts/prometheus-statsd-exporter/README.md:64]: Not sure if we still need to mention it as it's now officially deprecated.

**Review by torstenwalter — APPROVED** [review]

File: charts/prometheus-statsd-exporter/README.md
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/README.md:9]:
```suggestion
- Kubernetes 1.16+ with Beta APIs enabled
- Helm 3+
```

File: charts/prometheus-statsd-exporter/README.md
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/README.md:23]: No need for this comment anymore 🙂
```suggestion
```

File: charts/prometheus-statsd-exporter/README.md
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/README.md:34]:
```suggestion
```

File: charts/prometheus-statsd-exporter/README.md
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/README.md:45]:
```suggestion
```

File: charts/prometheus-statsd-exporter/README.md
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/README.md:56]:
```suggestion
```

File: charts/prometheus-statsd-exporter/Chart.yaml
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/Chart.yaml:8]: For Artifact Hub. This link is already used for `home`, let's just add the chart source so people know where to find it. If there are other relevant source links we can add them here too, and label them accordingly.
```suggestion
  - https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-statsd-exporter
annotations:
  "artifacthub.io/links": |
    - name: Chart Source
      url: https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-statsd-exporter
```

File: charts/prometheus-statsd-exporter/values.yaml
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/values.yaml:12]: Default tag should be pinned.
```suggestion
  tag: v0.18.0
```

**Review by scottrigby — CHANGES_REQUESTED** [review]

**Comment by scDisorder** [comment]
kkay, I should wait probably

**Review by scottrigby — APPROVED** [review]

File: charts/prometheus-statsd-exporter/templates/ingress.yaml
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/templates/ingress.yaml:8]: One more thing, we don't need this check if a prerequisite of this chart is
> Kubernetes 1.16+ with Beta APIs enabled

```suggestion
apiVersion: networking.k8s.io/v1beta1
```

**Review by scottrigby — CHANGES_REQUESTED** [review]

**Comment by scDisorder** [comment]
should we add `networking.k8s.io/v1` instead of `v1beta` for Kubernetes 19+ ?

**Comment by scDisorder** [comment]
> should we add `networking.k8s.io/v1` instead of `v1beta` for Kubernetes 19+ ?

well, I think this is more about helm backwards compatibility actually

**Comment by scDisorder** [comment]
woah, missclick

**Review by scottrigby — DISMISSED** [review]
This lgtm. If you want to do a semver check for 1.19+ and switch to that version you are welcome to do that. This version is still supported though in 1.19 so up to you.

**Review by monotek — APPROVED** [review]
@scottrigby 
any reason this was not merged?

**Comment by scottrigby** [comment]
@monotek I was just being patient waiting to see if they wanted to make a change based on my reply to their semver question.

**Comment by monotek** [comment]
ah, ok. i was just wondering, because it was approved and not merged. so i wont merge it too :D

**Comment by scDisorder** [comment]
not sure if default pathType must be `Prefix`, it is common thing to use `ImplementationSpecific` but i see no reasons up here

**Comment by scDisorder** [comment]
1037d952b8ecfe75

File: charts/prometheus-statsd-exporter/templates/ingress.yaml
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/templates/ingress.yaml:38]: @scDisorder I'm not really familiar with `pathType` yet apart from reading the docs, but there it says `ImplementationSpecific` is the default. Should this default follow, or is there a reason we want `Prefix` here?

File: charts/prometheus-statsd-exporter/values.yaml
Comment (scottrigby) [inline charts/prometheus-statsd-exporter/values.yaml:89]: See pathType default comment above

File: charts/prometheus-statsd-exporter/templates/ingress.yaml
Comment (scDisorder) [reply charts/prometheus-statsd-exporter/templates/ingress.yaml]: wow, probably i misudnerstood the doc or took it at the wrong time
fixed

**Review by scottrigby — APPROVED** [review]
lgtm 👏 nice work

**Pull request merged**
